pants-plugins/pants_explorer/server/uvicorn.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
This file was just a move. not sure why GitHub is showing it in full

Reply:
because the original is not removed.